ui/desktop/app/styles/app.scss

```diff
@@ -662,3 +662,29 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   gap: 0.5rem;
   width: inherit;
 }
+
+// Settings
+.settings-card {
```

Comment:
I am not sure we need an specific class. What do you think on giving an id to the whole settings page? then we can select #settings > hds card. That way the CSS will apply to every hds card element within settings, which I think is what we want. Thoughts?

Reply:
this needs to be a `classname`, because this component will be re-used in building out the rest of the cards.

Reply:
What I mean is if the main element or [the div that wraps the page body](https://github.com/hashicorp/boundary-ui/pull/2409/files#diff-23bd88f09a10c21b4b497fa14821c75c0276e1e2553634c6dee953e4a78804deR6) we add an id such as `#settings` or `#settings-section` then, you do not need to add a classname to every card we will add, as we do here. You can reference this on the css as `#settings > hds_card` (or the default class HDS generated for hds_card). And those styles will apply to all the HDS cards that are within settings section, without need for `classname`

All good with the current implementation, just anticipating this section will be several cards, I though we could reduce classname usage.

Reply:
agreed, I removed the classname from here and moved it to the wrapper. since we don't use `id` in our codebase for style, I kept it as `classname`. But I'm not opposed to using ID and this could be a team discussion on what the team prefers. adding to tech time

Reply:
Awesome @DhariniJeeva thanks for the clarification and bringing this topic to tech time 😃